kolibri/plugins/learn/assets/src/views/LibraryPage.vue

```diff
@@ -29,13 +29,15 @@
             :color="$themeTokens.text"
             :tooltip="$tr('viewAsList')"
             @click="toggleCardView('list')"
+            :disabled="currentViewStyle === 'list' && true"
```

Comment:
what is the purpose of ` && true`?

Reply:
whenever **currentVIewStyle** is a list, it returns a bool (**true**) to `:disabled` 
when the `:disabled` get **true** the button gets disabled.
same for the card view.

Reply:
Sorry, its useless I will change it

Reply:
@indirectlylit, I committed the changes and the result is still the same.